package.json

```diff
@@ -10,6 +10,7 @@
     "@types/node": "^12.12.26",
     "@types/react": "^16.9.19",
     "@types/react-dom": "^16.9.5",
+    "flexboxgrid": "^6.3.1",
```

Comment:
@raymondjosephsotto, we should use styled-components css instead of another dependency. You can do anything you can do with css in styled-components.